src/Perl6/Actions.nqp

```diff
@@ -980,6 +980,24 @@ register_op_desugar('time_n', -> $qast {
         }
     });
 }
+{
+    my $is_moar;
+    register_op_desugar('p6attrinited', -> $qast {
```

Comment:
OOC, shouldn't that have a non "p6-" prefix name?

Reply:
All of the desugars currently follow that convention still. Could do a pass through tweaking them at some point, though some will go away during RakuAST anyway.